pom.xml

```diff
@@ -7,29 +7,30 @@
     <relativePath />
   </parent>
 
-    <artifactId>script-security</artifactId>
-    <version>${revision}${changelist}</version>
-    <packaging>hpi</packaging>
-    <name>Script Security Plugin</name>
-    <description>Allows Jenkins administrators to control what in-process scripts can be run by less-privileged users.</description>
-    <url>https://wiki.jenkins.io/display/JENKINS/Script+Security+Plugin</url>
-    <properties>
-      <revision>1.63</revision>
-      <changelist>-SNAPSHOT</changelist>
-      <jenkins.version>2.60.3</jenkins.version>
-      <java.level>8</java.level>
-    </properties>
-    <licenses>
-        <license>
-            <name>MIT License</name>
-            <url>https://opensource.org/licenses/MIT</url>
-        </license>
-    </licenses>
-    <scm>
-        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-        <tag>${scmTag}</tag>
+  <artifactId>script-security</artifactId>
+  <version>${revision}${changelist}</version>
+  <packaging>hpi</packaging>
+  <name>Script Security Plugin</name>
+  <description>Allows Jenkins administrators to control what in-process scripts can be run by less-privileged users.</description>
+  <url>https://wiki.jenkins.io/display/JENKINS/Script+Security+Plugin</url>
+  <properties>
+    <revision>1.62</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>2.60.3</jenkins.version>
+    <java.level>8</java.level>
+    <configuration-as-code.version>1.15</configuration-as-code.version>
```

Comment:
```suggestion
    <configuration-as-code.version>1.30</configuration-as-code.version>
```

Reply:
+1, it was the latest at the time of creation